server/connection/socket.js

```diff
@@ -0,0 +1,93 @@
+const Websocket = require('ws');
+const {
+  createChat,
+  getReciveChatRoomInfo,
+  plusSellerUnreadCount,
+  plusBuyerUnreadCount,
+  updateLastChat,
+} = require('../data/chat');
+
+const rooms = new Map();
+
+class Socket {
+  constructor(server) {
+    this.wss = new Websocket.Server({
+      server,
+      verifyClient: async (info, cb) => {
+        const parsedUrl = info.req.url.split('/');
+        const [productId, buyerId, sellerId] = [parsedUrl[2], parsedUrl[3], parsedUrl[4]];
```

Comment:
parsedUrl을 아래에서 사용하지 않는다면 다음같이 수정할 수도 있겠네요 :)
```suggestion
        const [productId, buyerId, sellerId]= info.req.url.split('/').slice(2);
```

Reply:
오 훨씬 깔끔하네요.. 감사합니다! 👍